src/main/webapp/chart.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>A New Chart</title>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+	<script type="text/javascript">
```

Comment:
Small nitpick but make sure to have both `<script>` tags in line with each other. Right now line 6 and line 32 `<script>` tags aren't in line.

Reply:
agreed, other than that everything looks good!